Microsoft.NET.Build.Containers/Image.cs

```diff
@@ -15,6 +17,8 @@ public class Image
 
     internal List<Layer> newLayers = new();
 
+    internal HashSet<Label> labels = new();
```

Comment:
We'll want to treat this like an init field - Labels inherit from the parent Image, and so in the Image constructor we'll want to grab the Labels property from the underlying image (which might be null) and initialize the HashSet from the JsonObject there.